Sources/Identity/ProductPaidPrice.swift

```diff
@@ -40,7 +40,7 @@ import Foundation
     ///   - currency: Currency paid
     ///   - amount: Amount paid
     ///   - formatted: Formatted price string with currency symbol
-    init(currency: String, amount: Double, formatted: String) {
+    public init(currency: String, amount: Double, formatted: String) {
```

Comment:
@facumenzella I deduced from the API tests and from the init's documentation that this was supposed to be public. Please, lmk if that's not right

Reply:
Done in https://github.com/RevenueCat/purchases-ios/pull/5638/commits/abdcf2eeade185f0b692d4a6e718630daf3bb7b3